src/stream-analytics/azext_stream_analytics/custom.py

```diff
@@ -0,0 +1,199 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-statements
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-locals
+# pylint: disable=unused-argument
+
+
+from azure.cli.core.commands.client_factory import get_mgmt_service_client
+from azure.cli.core.profiles import ResourceType
+EXPANDED_FIELDS = ['inputs', 'transformation', 'outputs', 'functions']
+EXPANDED_ALL_STRING = ','.join(EXPANDED_FIELDS)
+
+
+def _get_resource_group_location(cli_ctx, resource_group_name):
+    client = get_mgmt_service_client(cli_ctx, ResourceType.MGMT_RESOURCE_RESOURCES)
+    # pylint: disable=no-member
+    return client.resource_groups.get(resource_group_name).location
+
+
+def create_stream_analytics_job(cmd, client, resource_group, name, location=None, tags=None,
+                                events_out_of_order_policy=None, output_error_policy=None,
+                                events_out_of_order_max_delay_in_seconds=None,
+                                events_late_arrival_max_delay_in_seconds=None, data_locale=None):
+
+    body = {
+        'location': location or _get_resource_group_location(cmd.cli_ctx, resource_group),
+        'tags': tags,
+        'properties': {
+            'sku': {'name': 'Standard'},
```

Comment:
hardcode sku name?

Reply:
yes, currently only `Standard` is available, that and this property is marked as required for REST api. So we need to hard code it here.

Reply:
what's type of sku? enum? if yes, we should expose sku as argument, with enum contains one value: Standard, maybe with default value

Reply:
yes, enum value defined in spec. I consider this before, however give it up as last. Because I think actually user can do nothing with this parameter. When more values are supported, manual works are still needed even if we add it as a parameter currently.

And another similar property is `compatibility_level`, in the rest spec, it says the only value is 1.0(set to 1.1 can succeed, 1.2 will failed. currently portal support 1.0, 1.1, 1.2). So I just emit these properties and leave them to the new api version support in the future.

Reply:
for `compatibility_level`, as it is not marked as required, service will give it a default value 1.0 if it is not in the request body.

Reply:
got it. it make sense when considering from implementation perspective. while from user perspective, user need to understand that which default sku is used. sku is key elemenet of pricing.

Reply:
`--sku-name` added.